package NYT.NTableServer.NProto;

import "yt_proto/yt/core/misc/proto/guid.proto";
import "yt/client/chunk_client/proto/data_statistics.proto";

import "yt/server/lib/tablet_server/proto/tablet_manager.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqSendTableStatisticsUpdates
{
    required int64 table_count = 1;
}

message TReqUpdateTableStatistics
{
    message TEntry
    {
        required NYT.NProto.TGuid table_id = 1;
        optional NChunkClient.NProto.TDataStatistics data_statistics = 2;
        reserved 3; // deprecated
        optional NTabletServer.NProto.TTabletResources tablet_resource_usage = 6;
        optional uint64 modification_time = 4;
        optional uint64 access_time = 5;
    }

    repeated TEntry entries = 1;
}
